pkg/event/event.go

```diff
@@ -23,11 +23,16 @@ import (
 	"github.com/redhat-cne/sdk-go/pkg/types"
 )
 
+const (
+	// EVENT is the generic event type
+	EVENT = "com.redhat.cne.event"
```

Comment:
this value is types as defined by API spec , and not to use redhat

Reply:
done